### PR TITLE
Lab1

### PR DESCRIPTION
**Pull request by conawt**
This will add the MATLAB live script and HTML files for LAB1. @jeffjiang08

### CODE REVIEW

**Comment by conawt** [comment]
Reviewed Jeff's comments. Incorporated some methods for examining the sign of sigma using complimentary expressions. Kept the histograms because they resemble discrete bins while the stairs still seem too curve-like, to me.

**Comment by conawt** [comment]
Ready to be graded. @oliviatessa

File: LAB1.html [hunk 1]
```diff
@@ -0,0 +1,200 @@
...
...UhCEpKQhCQkIQlJSEISkpCEJCQhCUlIQhKSkIQkJCEJSUhCEpKQhCQkIQlJSEISkpCEJCQhCUlIQhKSkIQkJCEJSUhCEpKQhCQkIQlJSEISkpCEJCQhCUlIQhKSkIQkJCEJSUhCEpKQhCQkIQlJSEISkpCEJCQhCUlIQhKSkIQkJCEJSUhCEpKQhCQkIQlJSEISkpCEJCQhCUlIQhKSkIQkJCEJSUhCEpKQhCQkIQlJSEISkpCEJCQhCUlIQhKSkIQkJCEJSUhCEpKQhCQkIQlJSEISkpCEJCQhCUlIQhKSkIQkJCEJSUhCEpKQhCQkIQlJSEISkpCEJCQhCUlIQhKSkIQkJCEJSUhCEpKQhCQkIQlJSEISkpCEJCQhCUlIQhKSkIQkJCEJSUhCEpKQhCQkIQlJSEISkpCEJCQhCUlIQhKSkIQkJCEJSUhCEpKQhCQkIQlJSEISkpCEJCQhCUlIQhKSkIQkJCEJSUhCEpKQhCQkIQlJSEISkpCEJCQhCUlIQhKSkIQkJCEJSUhCEpKQhCQkIQlJSEISkpCEJCQhCUlIQhKSkIQkJCEJSUhCEpKQhCQkIQlJSEISkpCEJCQhCUlIQhKSkIQkJCEJSUhCEpKQhCQkIQlJSEISkpCEJCQhCUlIQhKSkIQkJCEJSUhCEpKQhCQkIQlJSEISkpCEJCQhCUlIQhKSkIQkJCEJSUhCEpKQhCQkIQlJSEISkpCEJCQhCUlIQhKSkIQkJCEJSUhCEpKQhCQkIQlJSEISkpCEJCQhCUlIQhKSkIQkJCEJSUhCEpKQhCQkIQlJSEISkpCEJCQhCUlIQhKSkIQkJCEJSUhCEpKQhCQkIQlJSEISkpCEJCQhCUlIQhKSkIQkJCEJSUhCEpKQhCQkIQlJSEISkpCEJCQhCUlIQhKSkIQkJCEJSUhCEpKQhCQkIQlJSEISkpCEJCQhCUlIQhKSkIQkJCEJSUhCEpKQhCQkIQlJSEISkpCEJCQhCUlIQhKSkIQkJCEJSUhCEpKQhCQkIQlJSEISkpCEJCQhCUlI4gtf+AJxYgoWqeXLlzMXRVHwbm677TaWLFlCvV7nWG644Qa+/e1vc++99/LRj36UM844g+7ubrZt28Z5553Hzp07OXDgAMfyy1/+EtvYxja2sY1tbGMb29jGNraxjW1sYxvb2MY2trGNbWxjG9vYxja2sY1tb
```
Comment (oliviatessa) [inline LAB1.html:44]: What is the normal distribution? Describe it and it's parameters

Comment (oliviatessa) [inline LAB1.html:80]: Again, explain what the lognormal distribution is

Comment (oliviatessa) [inline LAB1.html:88]: Title, legend, axis labels - plots should be self-contained so that I could look at only them and understand what they are

Comment (oliviatessa) [inline LAB1.html:106]: Explain why you adjusted these parameters. Why do they change the distribution?

Comment (oliviatessa) [inline LAB1.html:133]: Why can you use norminv() for a lognormal distribution?

Comment (oliviatessa) [inline LAB1.html:140]: What could be the reason for this? Go into more detail

Comment (oliviatessa) [inline LAB1.html:162]: I think your statistical question could be reworded to be more clear.

Comment (oliviatessa) [inline LAB1.html:198]: Good discussion, I'd like to see more like this throughout the lab.

**Comment by oliviatessa** [comment]
Overall, your lab writeup could use more discussion. Remember you are writing as if to explain to someone who does not have a background in statistics or your work. Explain with words what you are doing and why you are doing. Including seemingly simple concepts such as integrating pdfs to get probabilities and sigmas.

**Pull request closed without merging**